Szymon/src/test/java/zadanie6/page/RegisterPage.java

```diff
@@ -0,0 +1,191 @@
+        driver.findElement(By.id("customer.address.city")).sendKeys(city);
+    }
+
+    public void enterState(String state) {
+        driver.findElement(By.id("customer.address.state")).sendKeys(state);
+    }
+
+    public void enterZipCode(String zipCode) {
+        driver.findElement(By.id("customer.address.zipCode")).sendKeys(zipCode);
+    }
+
+    public void enterPhoneNumber(String phoneNumber) {
+        driver.findElement(By.id("customer.phoneNumber")).sendKeys(phoneNumber);
+    }
+
+    public void enterSsn(String ssn) {
+        driver.findElement(By.id("customer.ssn")).sendKeys(ssn);
+    }
+
+    public String getRandomUsername(int numberOfChars) {
+        String randomString = RandomStringUtils.randomAlphanumeric(numberOfChars);
+        return "user_" + randomString;
+    }
+
+    public void enterUsernameToRegisterForm(String usernameIncoming) {
+        driver.findElement(By.id("customer.username")).sendKeys(usernameIncoming);
+    }
+
+    public void enterPasswordToRegisterForm(String password) {
+        driver.findElement(By.id("customer.password")).sendKeys(password);
+    }
+
+    public void confirmPasswordField(String password) {
+        driver.findElement(By.id("repeatedPassword")).sendKeys(password);
+    }
+
+    public String getConfirmMessageText() {
+        return driver.findElement(By.xpath("//div[@id='rightPanel']")).getText();
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
+
+    public String getAddres() {
+        return addres;
+    }
+
+    public void setAddres(String addres) {
+        this.addres = addres;
+    }
+
+    public String getCity() {
+        return city;
+    }
+
+    public void setCity(String city) {
+        this.city = city;
+    }
+
+    public String getState() {
+        return state;
+    }
+
+    public void setState(String state) {
+        this.state = state;
+    }
+
+    public String getZipCode() {
+        return zipCode;
+    }
+
+    public void setZipCode(String zipCode) {
+        this.zipCode = zipCode;
+    }
+
+    public String getSsn() {
+        return ssn;
+    }
+
+    public void setSsn(String ssn) {
+        this.ssn = ssn;
+    }
+
+    public String getFirstName() {  //wiem, że nie tak
+        return firstName;
+    }
+
+    private void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+    public String getPhoneNumber() {
+        return PhoneNumber;
+    }
+
+    public void setPhoneNumber(String phoneNumber) {
+        PhoneNumber = phoneNumber;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername() {
+        username = getRandomUsername(5);
+    }
+
+    public void createRegularUser() {
```

Comment:
Tak jak mowilem - cala ta metoda nie do konca pasuje do koncepcji interfejsu programistycznego dla strony.
Skladanie scenariuszy testowych to roobota testow, a nie stron samych w sobie.